digits/model/views.py

```diff
@@ -96,6 +95,21 @@ def customize():
             'snapshot': snapshot
             })
 
+@blueprint.route('/extension-static/<extension_id>/<path:filename>')
```

Comment:
There can be a data and a view extension with the same name, so it needs to be disambiguated. 

You are right that it should move to digits/views.py, but I'll need to add a view/data segment to the path.